core/src/main/scala/org/apache/pekko/persistence/jdbc/query/dao/ReadJournalQueries.scala

```diff
@@ -29,6 +29,9 @@ class ReadJournalQueries(val profile: JdbcProfile, val readJournalConfig: ReadJo
   val messagesQuery = Compiled(_messagesQuery _)
   val eventsByTag = Compiled(_eventsByTag _)
   val journalSequenceQuery = Compiled(_journalSequenceQuery _)
+  val lastPersistenceIdSequenceNumberQuery = Compiled {
+    _lastPersistenceIdSequenceNumberQuery _
```

Comment:
same - please don't change the code style - look at the line above for an indication of style

Reply:
Sure! Tried to align to existing code style in <https://github.com/apache/pekko-persistence-jdbc/pull/267/commits/49b7262dbcfe47279d580537a7c65dcdfd460fc8>.